back/engines/commercial/insights/app/services/insights/front_end_format_text_network.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Insights
+  # = FrontEndFormatTextNetwork
+  #
+  # This class implements a representation for text networks that is convenient
+  # to work with for the front-end.
+  class FrontEndFormatTextNetwork
+    attr_reader :id
+
+    # @param [Insights::View] view
+    def initialize(view)
+      @id = "network-#{view.id}"
+      @network = NLP::TextNetwork.merge(
+        # Namespacing networks wrt to the language to avoid id collisions.
```

Comment:
It's to avoid collisions between node identifiers in the FE representation of the network, not between record identifiers in the DB. The NLP service is identifying the node using the keyword. But in a multilingual context, the same keyword could appear in different languages and then they are no longer unique (a bit annoying for an identifier). 

Merging those nodes would have been another option, but:
- there is no guarantee that the same word in different languages refer to the same concept;
- it would make a lot of sense to combine ex post pagerank scores (importance score computed by the nlp service). 

So I opted to keep multiple nodes with the same keyword but different identifiers.